response_operations_ui/templates/collection-exercise.html

```diff
@@ -173,10 +175,17 @@ <h2 class="venus u-pt-m">Collection instruments</h2>
         <div class="u-mb-m">
           <form id="form-load-ci" action="" class="form" method="post" enctype="multipart/form-data">
             <p>
-              {{ 'Add another collection instrument (CI)' if collection_instruments else 'Choose a collection instrument (CI) to load'}}
+              Add {{'another' if collection_instruments else 'a'}} collection instrument. Must be XLSX
             </p>
-            <label class="upload-file u-vh" for="ciFile">Choose CI file</label>
-            <input id="ciFile" type="file" class="" name="ciFile" accept=".xlsx">
+            <div id="ciFileErrorPanel" class="{{'panel panel--simple panel--error' if error.section == 'ciFile' else ''}}">
+              <div id="ciFileErrorPanelBody" class="{{'panel__body' if error.section == 'ciFile' else ''}}">
+                <p id="ciFileErrorText" class="hidden">
+                  Incorrect file type. Please choose a file type XLSX
```

Comment:
"Please choose a file *of* type XLSX"?

Reply:
The current text matches the designs, I'll pass it back by @nicprice first though